README.md

```diff
@@ -32,55 +32,42 @@ There are 2 main methods to install and run Gatling:
 
 The installation steps are:
 1. Install a JDK, Gatling needs at least version 8
-2. [Download](https://gatling.io/download/) and extract Gatling, these test plans are written for version 3.
-3. Copy or symlink the `src/test/scala/govuk` directory of this repository to `$GATLING_HOME/user-files/simulations/`
-4. Copy or symlink the `src/test/resources/test-data` directory of this repository to `$GATLING_HOME/user-files/resources/`
-
-#### Tips
-
-1. Setting up the symlink
-   NOTE: Don't copy the directory if you set up the symlink as this may throw errors
-    ```
-    $ ln -s /path/to/target /path/to/gatling/folder/
-    ```
-
-    A symlink is required for Gatling to access the simulation files in this repo:
-    ```
-    $ ln -s /Users/username/govuk/govuk-load-testing/src/test/resources/test-data /Users/username/gatling/user-files/resources/
-    ```
-
-    Check the symlink has been set up correctly:
-    ```
-    $ ls -la /Users/username/gatling/user-files/simulation
-    lrwxr-xr-x  1 username  staff  65 14 Feb 13:54 /Users/username/gatling/user-files/simulation -> /Users/username/govuk/govuk-load-testing/src/test/scala/govuk/
-
-    ```
-
-2. In `$GATLING_HOME/user-files/simulations/test-plans` delete any duplicate .scala files as they should now come from the symlink.
+1. Clone this repository into your `~/govuk/` directory
+1. [Download](https://gatling.io/download/) and extract Gatling, these test plans are written for version 3
+1. Rename and move the extracted Gatling directory to `~/govuk/gatling` to make it available in `/var/govuk/gatling` in your Virtual Machine
+1. Symlink the `src/test/resources/test-data` directory of this repository to `$GATLING_HOME/user-files/resources/`
+  ```
+  $ ln -s ~/govuk/govuk-load-testing/src/test/resources/test-data ~/govuk/gatling/user-files/resources/
+  ```
+1. Set the needed environment variables:
+  - `$ export 'GATLING_USERNAME=<test-user-email>'`
```

Comment:
the username and password will only be needed for test plans using Signon, so not the Frontend plan, is it worth mentioning that here?

Reply:
You have made it very clear in the `configuration options` section, maybe we could say something like 
``` 1. Set the needed environment variables as detailed in Configuration Options
for example:
   - `$ export 'GATLING_USERNAME=<test-user-email>'`
    - `$ export 'GATLING_PASSWORD=<test-user-password>'`
```
?

Reply:
I had troubles when trying to follow this without this piece of information. I think it's worth mentioning it here, before the actual command to run the test.
How about 
```
- `$ export 'GATLING_USERNAME=<test-user-email>'` (only for test plans using Signon)
```
?

Reply:
I also wonder if it's worth mentioning here these are credentials for the test user that has been set up in Signon in the environment it's going to test in e.g. staging.

Reply:
Yes!